### DISCUSSION TITLE
Welcome, Chris!

### DISCUSSION THREAD

**Discussion by cmstokoe** [ParanoidUser/yolo#1058] [Subscribe]
I can't help but feel I'm just the person to stack some beer on the wall

**Comment by ParanoidUser** [comment]
Ahoy, @cmstokoe!

Thank you for you interest in the project! We're very excited to have you as a collaborator. If you find any, feel free to post interesting general knowledge riddles. The invitation to join the collaborator group has been sent to you. Welcome aboard! 🚢

Cheers! 🍻
